ios/FluentUI/Command Bar/CommandBar.swift

```diff
@@ -8,10 +8,9 @@ import UIKit
 /**
  `CommandBar` is a horizontal scrollable list of icon buttons divided by groups.
  Provide `itemGroups` in `init` to set the buttons in the scrollable area. Optional `leadingItem` and `trailingItem` add fixed buttons in leading and trailing positions. Each `CommandBarItem` will be represented as a button.
- Set the `delegate` property to determine whether a button can be selected and deselected, and listen to selection changes.
  */
 @objc(MSFCommandBar)
-open class CommandBar: UIView {
+public class CommandBar: UIView, TokenizedControlInternal {
```

Comment:
I don't think the current main CommandBar has tokens?

Reply:
You're right, I'll remove all this.